Sources/Testing/Testing.docc/MigratingFromXCTest.md

```diff
@@ -355,6 +355,12 @@ their equivalents in the testing library:
 | `try XCTUnwrap(x)` | `try #require(x)` |
 | `XCTFail("…")` | `Issue.record("…")` |
 
+To check that two numeric values are equivalent within a specified
+accuracy, as in the analog of 
+[`XCTAssertEqual(_:_:accuracy:_:file:line:)`](https://developer.apple.com/documentation/xctest/3551607-xctassertequal),
+import [swift-numerics](https://github.com/apple/swift-numerics)
+and use the ``isApproximatelyEqual()`` function.
```

Comment:
Drop "currently"—we're not planning to add an equivalent precisely because swift-numerics is the preferred solution. We don't anticipate a change here.

"an equivalent operation for" -> "an equivalent of"

"you can use" -> "use"?